packages/create-sitecore-jss/src/templates/nextjs/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We should add the same comments we had back as well

Reply:
added